### PR TITLE
Rerun

### PR DESCRIPTION
**Pull request by nthomsencph**
Tl;DR: Some changes were needed to make the code run. Refactoredthe code for efficiency and readability. See pull request and running notes below.
I’ve taken the liberty of creating a new ‘avgBatch’ script that uses pandas to calculate summary statistics instead. See notebook and the new file. 

1)  The program doesn’t run. The .json files specified in ‘examples1’ are not in the ‘/data’ directory. Are we missing some data? Created a mechanism that matches files specified in examples1 with existing .json files in /data. 

.json files specified in ‘examples1’
2Sent.json
1Sent.json
1SentAfter.json
1SentSubord.json
1SentSubordBackwards.json
2Actions.json

.json files in /data
PastedGraphic-1.png

2) File paths in line ellipsesBatch.py::51 doesn’t account for the fact that the .json files are in the subdirectory ‘/data’. Updating. (Update: I see that it works with the .pl script :-) )

3) ‘data/2Actions.json’ is incompatible with the code. It doesn’t have an ‘A’ (answer) key. Therefore, it fails in ellipsesBatch.py::82. Discarding in my remaining test runs.

4) Refactored a lot of stuff. Left the core algorithm untouched.

5) Added support for iterations directly in Python

6) Refactored README

7) Added a config.cfg file to store the OpenAI API key in.

### CODE REVIEW

**Comment by nthomsencph** [comment]
This is my take on it. Hope it's useful.

**Review by nthomsencph — COMMENTED** [review]
@DanHardtDK need you to review this

**Review by DanHardtDK — APPROVED** [review]
